src/parser.js

```diff
@@ -84,6 +106,32 @@ function sliceCode(code, start, end) {
     return splitted.slice(start - 1, end).join('\n');
 }
 
+export function markersSliceCode( code, markername )
+{
+    if( markername === '' ) {
+        return code;
+    }
+    else {
+        // various language comment
+        var commentOpen="(\/+\/+|#|%|\/\\*)";
+        var commentClose="(\\*\/)?";
+        var balise="\\["+markername+"\\]";
+        var pattern=commentOpen + ".*\\s*" + balise + ".*\\s*" + commentClose;
+        var regstr=pattern+"([\\s\\S]*)"+pattern;
+        var reg = new RegExp(regstr);
```

Comment:
:memo: https://github.com/jonschlinkert/parse-comments parse code comments any lang.
It will be improved in the future.